ttnn/cpp/pybind11/operations/core.hpp

```diff
@@ -32,7 +32,9 @@ void py_module(py::module& module) {
 
     module.def(
         "reshape",
-        [](const ttnn::Tensor& tensor, const std::array<int32_t, 2>& shape) -> ttnn::Tensor { return ttnn::reshape(tensor, shape); },
+        [](const ttnn::Tensor& tensor, const std::array<int32_t, 2>& shape) -> ttnn::Tensor {
```

Comment:
Can you remove the lambda and use `&ttnn::reshape`?

Reply:
Only by using something like this to resolve the overload.  This looks uglier to me.
static_cast<ttnn::Tensor (*)(const ttnn::Tensor&, const ttnn::Shape&)>(&ttnn::reshape),